pkg/notifications/client.go

```diff
@@ -14,6 +14,7 @@ import (
 
 func SendNotification(orgId string, eventName EventName, repos []repositories.Repositories) {
 	if config.Get().NotificationsClient != nil {
+		log.Warn().Msgf("Notification started: %t", config.Get().NotificationsClient)
```

Comment:
%t is actually just true or false, you might use %v

Reply:
Is there a list of these somewhere, couldn't find them in the docs for zerolog